package.json

```diff
@@ -76,7 +78,8 @@
     ]
   },
   "pkg": {
-    "assets": "proto/*"
+    "assets": "proto/*",
+    "scripts": "node_modules/vm2/lib/setup-sandbox.js"
```

Comment:
Why do we need this? 

This library is deprecated and, if we really need it, we should use `isolated-vm`:
![image](https://github.com/grafana/grafana-image-renderer/assets/35176601/dbac8bf3-dde8-4f28-b6a9-569aad668092)

Reply:
Apparently this is required for the latest puppeteer to work. See [here](https://github.com/vercel/pkg/issues/1947). I'll look into replacing it with isolated-vim. Afaik it's only necessary during build, so can be also moved to dev dependencies.

Reply:
It looks like this dependency [has been removed](https://github.com/puppeteer/puppeteer/releases/tag/browsers-v1.4.5) in the latest version of `@puppeteer/browsers`, so it's not necessary anymore.